src/Microsoft.Azure.WebJobs.Host/Executors/JobHostConfigurationExtensions.cs

```diff
@@ -515,6 +522,41 @@ private static Assembly GetHostAssembly(IEnumerable<MethodInfo> methods)
             }
         }
 
+        #region Hackery for ExtensionLocator
```

Comment:
perhaps we should link to an issue if we have one? Hopefully the hackery is temporary?